### DISCUSSION TITLE
[jikong] normal connect failed (Device with address C8:47:8C:E5:BA:O2 was not found.), connecting with scanner

### DISCUSSION THREAD

**Discussion by setmel** [fl4p/batmon-ha#93] [Q&A]
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
20:49:41 INFO [bt] BT Discovery:
20:49:46 INFO [bt] BT Device   JK-B2A24S15P   address=C8:47:8C:E5:BA:02
20:49:46 INFO [bt] BMS JKBt(C8:47:8C:E5:BA:O2) keep alive enabled
20:49:46 INFO [main] connecting mqtt setmel@core-mosquitto
20:49:46 INFO [main] Fetching 1 BMS + 0 others concurrently, period=1.00s, keep_alive=True
20:49:46 INFO [sampling] connecting bms JKBt(C8:47:8C:E5:BA:O2)
20:49:46 INFO [main] mqtt watchdog loop started with timeout 900.0s
**20:49:50 INFO [jikong] normal connect failed (Device with address C8:47:8C:E5:BA:O2 was not found.), connecting with scanner**

What can happen?

**Comment by rdstevens** [comment]
Have you set up a Config entry for that device?

I think you might have got the last part of the address wrong: O2 should be 02 (I.e. zero-two, not letter-oh-two).

**Comment by setmel** [comment]
Efectivamente, me di cuenta al salir un mensaje diciendo que las letras
llegan hasta "F".
Funcionando perfectamente.
Muchas gracias!
Saludos.
